docs/api-guide/schemas.md

```diff
@@ -145,6 +145,21 @@ May be used to pass a canonical URL for the schema.
         url='https://www.example.org/api/'
     )
 
+#### `urlconf`
+
+String representing import path to the URL conf that you want
+to generate an API schema for. This defaults to the value of Django's
+ROOT_URLCONF setting.
+
+Useful for `Django multihost snippet`
+https://code.djangoproject.com/wiki/MultiHostMiddleware
```

Comment:
Exactly as I said in initial review...

"A string representing the import path to the URL conf that you want to generate an API schema for. This defaults to the value of Django's ROOT_URLCONF setting."

Only remaining issue is that some of the original text is still remaining.